apm-collector/apm-collector-remote/collector-remote-grpc-provider/pom.xml

```diff
@@ -55,14 +55,6 @@
             </extension>
         </extensions>
         <plugins>
-            <plugin>
```

Comment:
this plugin is setup in the root pom and inherited, this is why I deleted the many times it was copy pasted